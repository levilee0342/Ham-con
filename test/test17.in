-23237
-9253
1367
-13062
-6854
-9038
-13168
-2728
-8115
-15306
-5102
-3100
