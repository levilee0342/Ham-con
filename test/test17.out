1367
-23237